docs/_articles/en/soql/soql-builder.md

```diff
@@ -11,9 +11,15 @@ During beta, you can build simple query statements that include:
 
 - FROM clause for only one sObject type
 - SELECT clause to pick fields from the selected sObject
+- WHERE clause to filter your data
 - ORDER BY clause with support for ASC, DESC, NULLS FIRST, and NULLS LAST
 - LIMIT clause
 
+**Beta Limitations:**
+
+- You can still run complex queries in SOQL Builder even if you see the Unsupported Syntax informational message.
+- WHERE clauses can be quite complex. SOQL Editor supports simple WHERE expressions. You can combine conditions using AND or OR, but not both. SOQL Editor supports only one level of nesting.
```

Comment:
I think we should replace the 2 occurences of "SOQL Editor" with "SOQL Builder".

Reply:
I don't think `SOQL Editor supports only one level of nesting.` is necessary. Since it is actually logically the same to have `(A AND B) AND (C AND (D AND E))` and `A AND B AND C AND D AND E`, the SOQL Builder can actually support both of those, regardless of number of levels of nesting. It is only when you mix AND's and OR's that things become too complex.

Reply:
Good catch, @praksb! Thanks. @jgellin-sf - if I remove "one level of nesting", is the rest of the explanation ok? Anything I should add?

Reply:
No I think that should do it.